README.md

```diff
@@ -60,8 +60,17 @@ You should now be able to use riscv64-unknown-elf-gcc and its cousins.
 ### Installation (Linux)
 
 To build the Linux cross-compiler, pick an install path.  If you choose,
-say, `/opt/riscv`, then add `/opt/riscv/bin` to your `PATH` now.  Then, simply
-run the following command:
+say, `/opt/riscv`, then add `/opt/riscv/bin` to your `PATH` now. 
+You can add this command via command line when adding your path.
+
+	$ export PATH = $ PATH: / opt / riscv / binopt / riscv / bin
+
+But your permission may be incomplete. So entering the .bashrc file ($ nano .bashrc);
+
+	export RISCV_INSTALL = / opt / riscv
+	export PATH = $ PATH: $ RISCV_INSTAL / bin
```

Comment:
There are a bunch of extra spaces here, and INSTAL is misspelled, so this won’t work.